ui/util/parse-data.js

```diff
@@ -28,13 +28,16 @@ const parseCsv = (data, filters = []) => {
   };
 
   // Get rows content
-  const getData = list =>
+  const getData = (list) =>
     list
-      .map(item => {
+      .map((item) => {
         const row = [];
         // Apply filters
         Object.entries(item).forEach(([key, value]) => {
-          if (!filters.includes(key)) row.push(value);
+          if (!filters.includes(key)) {
+            const sanitizedValue = '"' + String(value).replaceAll('"', '\\"') + '"';
```

Comment:
Everything I read about escaping " with csv says to do "" . What's the reasoning for \\" ? Is that widely supported in inputs?